tests/test_data/missing_eval.json

```diff
@@ -13,7 +13,8 @@
         "name": "FastGradientMethod"
     },
     "dataset": {
-        "name": "mnist"
+        "module": "armory.data.datasets",
+        "name": "mnist_data"
```

Comment:
Little nit: can we make the `mnist_data` and `cifar10_data` methods (and the references to them) `mnist` and `cifar10`? It's a bit more intuitive and `datasets.mnist_data` feels a bit redundant.